### PR TITLE
[Client Hints] Address Genart review comments

### PR DESCRIPTION
**Pull request by yoavweiss**

### CODE REVIEW

**Comment by yoavweiss** [comment]
Friendly ping! :)

File: draft-ietf-httpbis-client-hints.md [hunk 1]
```diff
@@ -86,17 +94,19 @@ Such techniques are expensive to setup and maintain, and are not portable across
...
+This document also defines a new response header, Accept-CH, that allows an origin server to explicitly ask that clients send these headers in requests.
 
 Client Hints mitigate performance concerns by assuring that clients will only send the request headers when they're actually going to be used, and privacy concerns of passive fingerprinting by requiring explicit opt-in and disclosure of required headers by the server through the use of the Accept-CH response header.
 
-This document defines Client Hints, a framework that enables servers to opt-in to specific proactive content negotiation features, adapting their content accordingly. However, it does not define any specific features that will use that infrastructure. Those features will be defined in their respective specifications.
+The document does not define specific usages of Client Hints. Such usages need to be defined in their respective specifications.
+
+One example of such usage is the User Agent Client Hints [UA-CH].
```
Comment (mnot) [inline draft-ietf-httpbis-client-hints.md:108]: Here and elsewhere, markdown references are with {{ }} , not [ ]

File: draft-ietf-httpbis-client-hints.md [hunk 2]
```diff
@@ -56,11 +56,19 @@ informative:
...
+    -
+      ins: Y. Weiss
+      name: Yoav Weiss
+      organization: Google
 
 
 --- abstract
 
-HTTP defines proactive content negotiation to allow servers to select the appropriate response for a given request, based upon the user agent's characteristics, as expressed in request headers. In practice, clients are often unwilling to send those request headers, because it is not clear whether they will be used, and sending them impacts both performance and privacy.
+HTTP defines proactive content negotiation to allow servers to select the appropriate response for a given request, based upon the client's characteristics, as expressed in request headers. In practice, clients are often unwilling to send those request headers, because it is not clear whether they will be used, and sending them impacts both performance and privacy.
```
Comment (mnot) [inline draft-ietf-httpbis-client-hints.md:71]: Here and elsewhere, changing "user agent" to "client" implies that the scope of discussion -- including requirements -- includes intermediaries as well as UAs. Is that intentional?

Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md:71]: That's a good point, I haven't considered intermediaries. One of the points raised in the review was to align on either "client" or "user agent". I guess it'd be more appropriate to align on the latter.

File: draft-ietf-httpbis-client-hints.md (hunk 1 above)
Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md:108]: Changed

File: draft-ietf-httpbis-client-hints.md (hunk 2 above)
Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md]: Aligned on user agent. Let me know if that works

**Comment by yoavweiss** [comment]
Friendly ping! :) PTAL?

**Comment by mnot** [comment]
LGTM

**Comment by yoavweiss** [comment]
Thanks for reviewing!! :)

**Pull request merged**